javascript/core/objects/converting-to-and-from-json.md

```diff
@@ -0,0 +1,78 @@
+---
+author: EwaTrzemzalska
+
+type: normal
+
+category: fundamental
+
```

Comment:
An insight (currently) requires a [`levels` field](https://enkidevs.github.io/guidelines/Insights-guidelines.html#levels-m) that is one or more of the following:

    beginner
    basic
    medium
    advanced

For this use case, it's ok to use your best judgment or whatever levels are common within other insights of the `core/objects` workout.

Reply:
Related: https://github.com/enkidevs/curriculum/wiki/Insight-Documentation#levels-deprecated

Reply:
For example:

```yaml
levels:
  - basic
  - advanced
  - medium
```

Reply:
I didn't include this, because of "deprecated". What does it mean in this context?

Reply:
Technically it's about to be deprecated but hasn't been yet :)